File: app/src/main/java/com/mithilakshar/mithilapanchang/holidaydatamodel.java
package com.mithilakshar.mithilapanchang;

public class holidaydatamodel {

    private String holidayName,holidayDesc,imageUrl;


    public holidaydatamodel() {

    }

    public holidaydatamodel(String holidayName, String holidayDesc, String imageUrl) {
        this.holidayName = holidayName;
        this.holidayDesc = holidayDesc;
        this.imageUrl = imageUrl;
    }

    public String getHolidayName() {
        return holidayName;
    }

    public void setHolidayName(String holidayName) {
        this.holidayName = holidayName;
    }

    public String getHolidayDesc() {
        return holidayDesc;
    }

    public void setHolidayDesc(String holidayDesc) {
        this.holidayDesc = holidayDesc;
    }

    public String getImageUrl() {
        return imageUrl;
    }

    public void setImageUrl(String imageUrl) {
        this.imageUrl = imageUrl;
    }
}

File: app/src/main/java/com/mithilakshar/mithilapanchang/holidaydatamodel.kt
package com.mithilakshar.mithilapanchang

class holidaydatamodel {
    var holidayName: String? = null
    var holidayDesc: String? = null
    var imageUrl: String? = null

    constructor()
    constructor(holidayName: String?, holidayDesc: String?, imageUrl: String?) {
        this.holidayName = holidayName
        this.holidayDesc = holidayDesc
        this.imageUrl = imageUrl
    }
}
